common/supplemental/coverageLevels.xml

```diff
@@ -924,5 +924,17 @@ For terms of use, see http://www.unicode.org/copyright.html
 		<coverageLevel value="modern" match="numbers/minimalPairs/caseMinimalPairs[@case='%anyAlphaNum']"/>
         <coverageLevel value="modern" match="numbers/minimalPairs/genderMinimalPairs[@gender='%anyAlphaNum']"/>
 
+		<pathGroups>
+			<pathGroup id="characterLabel">
+				<pathMatch match="characterLabels/characterLabelPattern[@type='%anyAttribute']"/>
+				<pathMatch match="characterLabels/characterLabelPattern[@type='%anyAttribute'][@count='%allPlurals']"/>
+			</pathGroup>
```

Comment:
this mirrors the `<coverageLevel>` lines for characterLabelPattern, but without the coverage level.  I'm trying to make these easy to build and maintain, and also to keep Locales.txt as simple as possible.